multicluster/controllers/multicluster/core/local_cluster_manager.go

```diff
@@ -18,6 +18,7 @@ package core
 
 import (
 	"context"
+
 	"k8s.io/klog/v2"
 
```

Comment:
remove this empty line

Reply:
It seems to be added by goimports command to splits the imports into 3 sections